cn_indian_payroll/cn_indian_payroll/page/payroll_configuratio/payroll_configuratio.js

```
@@ -735,590 +32,392 @@ frappe.pages['payroll-configuratio'].on_page_load = function(wrapper) {
-							
+										frappe.call({
+                                        method: "cn_indian_payroll.cn_indian_payroll.overrides.payroll_configuration.get_salary_component",
+                                        args: {
+                                            data: rowData,
+                                            component: values.salary_components_table,
+                                            custom_field: values.custom_field_table,
+                                        },
+										callback:function(response)
+										{
 
+											 if(response && response.exc) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)